pipeline-yaml_5b5744d40428631d7a8940a9.md

```diff
@@ -615,6 +614,53 @@ were also defined on the `task` level.
 	          - echo $VAR_1
 	          - echo $APP_ENV
 
+### matrix
+
+The `martix` property allows you to define one or more environment variables
+with multiple values. After that, it creates multiple jobs by calculating the
+Cartesian product of all the sets of user defined values.
+
+So, the final outcome of the `matrix` property is concurrent jobs with the
+commands that are defined in the respective `jobs` property but different
```

Comment:
"but different environment variables that are defined in the `matrix` property."
=>
joined with all possible combinations of environment variables defined in the `matrix` property.

Reply:
Done.